test/metabase/api/public_test.clj

```diff
@@ -1550,56 +1550,90 @@
-                                                                     :target [:dimension (mt/$ids $orders.user_id->people.state)]
-                                                                     :value  ["CA" "WA"]}]))]
-                      (is (=? {:status "completed"}
-                              result))
-                      (testing "row_count isn't included in public endpoints"
-                        (is (nil? (:row_count result))))
-                      (is (= 6 (count (get-in result [:data :cols]))))
-                      (let [rows (mt/rows result)]
-                        (is (= 80 (count rows)))
-                        (is (= ["CA" "Affiliate" "Doohickey" 0 16 48] (first rows)))
-                        (is (= [nil "Google" "Gizmo" 1 52 186] (nth rows 50)))
-                        (is (= [nil nil nil 7 1015 3758] (last rows)))))))))))))))
+                 (testing "row_count isn't included in public endpoints"
+                   (is (nil? (:row_count result))))
+                 (is (= 6 (count (get-in result [:data :cols]))))
+                 (let [rows (mt/rows result)]
+                   (is (= 1144 (count rows)))
+                   (is (= ["AK" "Affiliate" "Doohickey" 0 18 81] (first rows)))
+                   (is (= ["CO" "Affiliate" "Gadget" 0 62 211] (nth rows 100)))
+                   (is (= [nil nil nil 7 18760 69540] (last rows))))))
+
+             (testing "with parameters"
+               (let [result (results :parameters (json/encode [{:name   "State"
+                                                                :id     "_STATE_"
+                                                                :slug   :state
+                                                                :target [:dimension (mt/$ids $orders.user_id->people.state)]
+                                                                :value  ["CA" "WA"]}]))]
+                 (is (=? {:status "completed"}
+                         result))
+                 (testing "row_count isn't included in public endpoints"
+                   (is (nil? (:row_count result))))
+                 (is (= 6 (count (get-in result [:data :cols]))))
+                 (let [rows (mt/rows result)]
+                   (is (= 80 (count rows)))
+                   (is (= ["CA" "Affiliate" "Doohickey" 0 16 48] (first rows)))
+                   (is (= [nil "Google" "Gizmo" 1 52 186] (nth rows 50)))
+                   (is (= [nil nil nil 7 1015 3758] (last rows)))))))))))))
+
+(deftest public-pivot-dashcard-errors-test
+  (testing "GET /api/public/pivot/dashboard/:uuid/dashcard/:dashcard-id/card/:card-id"
+    (testing "Shouldn't be able to execute a public DashCard if public sharing is disabled"
+      (mt/test-drivers (api.pivots/applicable-drivers)
+        (do-with-temp-dashboard-and-public-pivot-card
+         (fn [dash card dashcard]
+           (mt/with-temporary-setting-values [enable-public-sharing false]
+             (is (= "An error occurred."
```

Comment:
A testing label would be nice here

Reply:
Good spotting, I rearranged some things and forgot to move the testing description. Updated in 19d280ece8f619c8992e2911b50a19c7a38f620e